flight/Modules/VibrationAnalysis/vibrationanalysis.c

```diff
@@ -52,7 +51,15 @@
 // Private constants
 
 #define MAX_QUEUE_SIZE 2
-#define STACK_SIZE_BYTES (200 + 484 + (13*fft_window_size)*0) // The fft memory requirement grows linearly 
+
+/*
+    Stack usage reported by gcc:
+    VibrationAnalysisCleanup     16
+    VibrationAnalysisInitialize  24
+    VibrationAnalysisStart       48
+    VibrationAnalysisTask        192
+*/
+#define STACK_SIZE_BYTES (192 + 48 + 16 + (2*3*window_size)*0) // The memory requirement grows linearly 
```

Comment:
I don't think this is the right way to calculate the stack size, because ... not all of these functions run at once.  Also, they call into other library functions that use stack. 

Probably best practice is to pick a small value, like 480, and then when the module is running, ensure in taskstats that at least 100 bytes is free.

I have some code to do static stack-depth analysis but it's pretty challenging to use, because we use function pointers so much.